rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -269,6 +269,13 @@
           "type": "string",
           "format": "email"
         }
+      },
```

Comment:
Add card wrapper to `order_contact` as well.

Reply:
added